ts/card-info/lib.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+import { Stats } from "../lib/proto";
+import { postRequest } from "../lib/postrequest";
+
+export async function getCardStats(cardId: number): Promise<Stats.CardStatsResponse> {
+    return Stats.CardStatsResponse.decode(
+        await postRequest("/_anki/cardStats", JSON.stringify(cardId))
```

Comment:
I'll push a follow-up patch that converts this to a dict instead of passing a bare argument, as it's easier to extend in the future.